test/e2e/generator/generate.go

```diff
@@ -39,6 +40,12 @@ var (
 		"kill":       0.1,
 		"restart":    0.1,
 	}
+	nodeMisbehaviors = weightedChoice{
+		misbehaviorOption{"double-prevote"}: 1,
+		misbehaviorOption{}:                 9,
+	}
+	// amount of blocks from startup before misbehavior occurs
+	misbehaviorHeighDelta = uniformChoice{9, 11, 13, 15}
```

Comment:
We don't need to use a preset list here, we can just do something like `startAt + 5 + r.Intn(10)`